jenkins/Jenkinsfile

```diff
@@ -0,0 +1,34 @@
+pipeline {
+
+    options {
+        disableConcurrentBuilds()
+        buildDiscarder(logRotator(numToKeepStr: '5'))
+        timeout(time: 5, unit: 'MINUTES')
+    }
+
+    agent any
+
+    triggers {
+        pollSCM('H/5 * * * *')
+    }
+
+    stages {
+        stage('Build') {
+            agent {
+                docker {
+                    reuseNode true
+                    image 'adoptopenjdk/maven-openjdk11:latest'
+                    args '-v $HOME/.m2:/root/.m2'
+                }
+            }
+            steps {
+                sh "mvn -B -U -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -DskipTests clean package"
```

Comment:
oooh interesting.  so this runs inside the docker agent specified above?  nice!  Why skipTests?

Reply:
That's right... yes, I skip them because there are failed tests

Reply:
where does the .jar file go?  not seeing it listed as an artifact.  also - maybe a separate process, but likely need a release tag in git.